assembly-syntax/src/testing/context.rs

```diff
@@ -22,7 +22,7 @@ use crate::{
 /// Some of the assertion macros defined in this crate require a [SyntaxTestContext], so be aware of
 /// that.
 pub struct SyntaxTestContext {
-    source_manager: Arc<dyn SourceManager + Send + Sync>,
+    source_manager: Arc<dyn SourceManagerSync>,
```

Comment:
*Question:* Does this actually need to be `SourceManagerSync` rather than `SourceManager` here? AFAIK all of the syntax-related tests are single-threaded and don't call into anything that requires a `Sync` source manager.

Reply:
That was a find-and-replace - fixed now